runtime/bcutil/CMakeLists.txt

```diff
@@ -71,6 +71,12 @@ target_link_libraries(j9dyn_sources
 # also add explicit dependency on running tracegen, because cmake wont pick this up outside of this dir
 add_dependencies(j9dyn_sources trc_j9bcu)
 
+if(OMR_OS_AIX)
```

Comment:
needs to limit it further down to avoid kicking in for OpenXL compiler (likely fail ... since -q option is not recognized there)

Reply:
Good catch; done.